python_modules/dagster-graphql/dagster_graphql_tests/test_cli.py

```diff
@@ -19,10 +19,14 @@ def dagster_cli_runner():
         with instance_for_test(
             temp_dir=dagster_home_temp,
             overrides={
+                "run_coordinator": {
+                    "module": "dagster._core.run_coordinator.immediately_launch_run_coordinator",
+                    "class": "ImmediatelyLaunchRunCoordinator",
+                },
```

Comment:
i wonder if this is common enough that it should be a flag on instance_for_test for ease of use purposes?